client/my-sites/promote-post-i2/components/campaign-item/index.tsx

```diff
@@ -77,15 +77,26 @@ export default function CampaignItem( props: Props ) {
 	);
 
 	let budgetString = '-';
+	let budgetStringMobile = '';
 	if ( is_evergreen && campaignDays ) {
-		budgetString = `$${ formatCents( totalBudget ) } weekly`;
+		/* translators: Daily average spend. dailyAverageSpending is the budget */
+		budgetString = sprintf(
+			/* translators: %s is a formatted amount */
+			'%s weekly',
```

Comment:
dont we need here a __() function.. 

is sprintf going to translate too??

Reply:
Yeah, that was my bad, check the last commit which I added the translate function.